README.md

```diff
@@ -58,6 +58,54 @@
 
 - [ ] Move engine maintainership to apps, review code [APPS]
 
+### Launch Unified Engine Playground [DevRel]
+
+- [ ] Define Engine Playground experience (sandbox for srcd)
+- [ ] Launch new Engine Playground Alpha Version
+- [ ] Add metrics from playground to main dashboard
+- [ ] Define main use cases and personas for the engine
+
+### source{d} Engine awareness & use cases [DevRel]
+
+- [ ] Define roadmap + timeline of Engine releases
+- [ ] Identify at least 3 companies effectively using the Engine 
+- [ ] Publish at least 1 case study
+- [ ] Press announcement for Engine releases 0.2 & power users
+- [ ] Define "Give me your end of year budget" campaign
+- [ ] Create enterprise workshop on Code as Data and run it at least once
+- [ ] Offer 6 companies our CodeAsData workshop and schedule at least one
+
+### Promote current source{d} stack [DevRel]
+
+- [ ] Develop strategy & app for the "What's in your code?" campaign
+- [ ] Develop strategy & app for the "Analyzing most popular OSS Projects" campaign
+- [ ] Write and promote blog series for each feature in the engine
+- [ ] source{d} Engine featured at 3 conferences
+- [ ] source{d} Engine featured at 3 meetups
+
+#### Run beta program on source{d} Engine [DevRel]
+
+- [ ] Identify at least 3 users of source{d} Engine with different personas/use cases
+- [ ] Meet regularly (biweekly) with users and gather feedback
+
+### Defined unified web client [DevRel]
+
+- [ ] Write specifications and draft design for a unified web client (instead of gitbase + bblfsh, etc)
+
+### source{d} Engine add-ons (applications) [DevRel]
```

Comment:
Maybe these should be under the top-level `# Bring 'Code as Data' to market with enterprises`?